app/javascript/gobierto_dashboards/modules/contracts_controller.js

```diff
@@ -1,3 +1,118 @@
+import Vue from "vue";
+import VueRouter from "vue-router";
+import { getRemoteData } from '../webapp/lib/get_remote_data'
+
+Vue.use(VueRouter);
+Vue.config.productionTip = false;
+
 export class ContractsController {
+  constructor(options) {
+    const selector = "gobierto-dashboards-contracts-app";
+
+    // Mount Vue applications
+    const entryPoint = document.getElementById(selector);
+
+    if (entryPoint) {
+      const htmlRouterBlock = `
+        <keep-alive>
+          <transition name="fade" mode="out-in">
+            <router-view :key="$route.fullPath"></router-view>
+          </transition>
+        </keep-alive>
+      `;
+
+      entryPoint.innerHTML = htmlRouterBlock;
+
+      const Home = () => import("../webapp/containers/shared/Home.vue");
+      const Summary = () => import("../webapp/containers/summary/Summary.vue");
+      const ContractsIndex = () => import("../webapp/containers/contract/ContractsIndex.vue");
+      const ContractsShow = () => import("../webapp/containers/contract/ContractsShow.vue");
+      const TendersIndex = () => import("../webapp/containers/tender/TendersIndex.vue");
+      const TendersShow = () => import("../webapp/containers/tender/TendersShow.vue");
+
+      const router = new VueRouter({
+        mode: "history",
+        routes: [
+          { path: "/dashboards/contratos", component: Home,
+            children: [
+              { path: "resumen", name: "summary", component: Summary },
+              { path: "contratos", name: "contracts_index", component: ContractsIndex },
+              { path: "contratos/:id", name: "contracts_show", component: ContractsShow },
+              { path: "licitaciones", name: "tenders_index", component: TendersIndex },
+              { path: "licitaciones/:id", name: "tenders_show", component: TendersShow },
+            ]
```

Comment:
@eltercero Don't you tell me you gonna add the props here?

Reply:
Yes, but I'll do it in the latest issue on development to avoid conflicts in three separate branches https://github.com/PopulateTools/gobierto/pull/3053#discussion_r424976360